src/renderer/components/settings-page/index.tsx

```diff
@@ -16,9 +16,8 @@ import { Select } from '../select';
 import { useAudioInputOutputDevices } from './hooks';
 
 export const SettingsPage: FC<{
-  onNext: MouseEventHandler<HTMLElement>;
-  onBack: MouseEventHandler<HTMLElement>;
-}> = ({ onNext, onBack }) => {
+  onSelect: MouseEventHandler<HTMLElement>;
+}> = ({ onSelect }) => {
```

Comment:
Rename this prop to be more descriptive.

Reply:
Also, why not just keep calling this `onBack`?

Reply:
onBack goes back to the previous page by using current page index - 1, since our page is not actually in the page list by sequence, we need to jump to a specific index